.darglint

```diff
@@ -0,0 +1,7 @@
+[darglint]
```

Comment:
Same as above. This is a docstring linter. ansible.network is just a meta collection with no python in it.